### ISSUE TITLE
Resizing cursor showed near caption button when IsResizable=false while ExtendsContentIntoTitleBar=true

### ISSUE THREAD

**Issue by HO-COOH** [microsoft/microsoft-ui-xaml#9670]
### Describe the bug

It will show a resizing cursor at the border near the caption button. Other part of the border will not show a resizing cursor.

### Steps to reproduce the bug

In the window constructor, use this code
```csharp
        (AppWindow.Presenter as OverlappedPresenter).IsResizable = false;
        ExtendsContentIntoTitleBar = true;
```

### Expected behavior

Don't show resizing cursor when the window is not resizable.

### Screenshots

![wrong-cursor](https://github.com/microsoft/microsoft-ui-xaml/assets/42881734/43d3eb8c-5c74-4e03-8dcd-8d21ffee8536)

### NuGet package version

WinUI 3 - Windows App SDK 1.5.3: 1.5.240428000

### Windows version

Windows 10 (1809): Build 17763

**Comment by 0x5bfa** [comment]
The same happens when you maximize the window.

**Comment by codendone** [comment]
The issue when the window is maximized is tracked by #8665. I believe that upcoming fix didn't address this IsResizable=false scenario.

**Comment by ghost1372** [comment]
i tested with v1.6-exp2 and does not fixed yet
```
ExtendsContentIntoTitleBar = true;
SetTitleBar(AppTitleBar);

((OverlappedPresenter)AppWindow.Presenter).IsResizable = false;
```

![xx](https://github.com/microsoft/microsoft-ui-xaml/assets/9213496/0a7a2fda-1ef9-4dd7-b2ee-b3f77c7f237e)

**Comment by codendone** [comment]
This was closed as fixed/duplicate of #7629. The fix will be in the next 1.6 release.
